doozer/doozerlib/cli/config_plashet.py

```diff
@@ -586,11 +588,12 @@ def from_tags(config: SimpleNamespace, brew_tag: Tuple[Tuple[str, str], ...], em
                 released_nvre_obj = released_package_nvre_obj[package_name]
 
             def is_embargoed(an_nvre):
-                # .p1 or inclusion in the embargoed_tag_nvrs indicates this rpm is embargoed OR *was* embargoed.
+                # .p1/.p3 or inclusion in the embargoed_tag_nvrs indicates this rpm is embargoed OR *was* embargoed.
                 # We can ignore it if it has already shipped.
                 test_nvre_obj = parse_nvr(an_nvre)
                 if released_nvre_obj is None or compare_nvr(test_nvre_obj, released_nvre_obj) > 0:  # If this nvr hasn't shipped
-                    if '.p1' in an_nvre or strip_epoch(an_nvre) in embargoed_tag_nvrs:  # It's embargoed!
+                    pflag = get_visibility_suffix(runtime.build_system, BuildVisibility.PRIVATE)
+                    if f'.{pflag}' in (an_nvre or '') or strip_epoch(an_nvre) in embargoed_tag_nvrs:  # It's embargoed!
```

Comment:
I think it would be nice to have a `is_nvr_embargoed` / `is_nvre_embargoed` convenience methods to make it easier for the caller to determine this